pyproject.toml

```diff
@@ -29,8 +29,10 @@ django-mirror = "^0.1.13"
 mammoth = "^1.7.1"
 tomli = "^2.0.2"
 django-ninja = "^1.3.0"
+cryptography = "^42.0.0"
 
-# sortable = "@3", check nofo_index.html
```

Comment:
Can I leave this here? 

It's a dependency we have in JS (we are loading it from a CDN) and I didn't want to lose track of it so I added this commented line here. 

Here's the line we care about: https://github.com/HHS/simpler-grants-pdf-builder/blob/main/bloom_nofos/nofos/templates/nofos/nofo_index.html#L97